coverage.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# We want to make sure the files of our dependencies are availiable so we can update their pragma.
+forge install
+
+# First, we change all Aave pragmas from 0.8.10 to 0.8.13, because this is the first solc version with via-ir which we  and we need
+# via-ir because importing the PoolConfigurator contract into the ATokenFlexVoting tests causes a stack too deep error.
+find lib/aave-v3-core/contracts -type f -name "*.sol" -exec awk -i inplace '{gsub(/pragma solidity 0\.8\.10;/, "pragma solidity 0.8.13;"); print}' {} \;
```

Comment:
I get a ton of warnings like this every time `awk` is invoked:
```
awk: can't open file {gsub(/pragma solidity 0\.8\.10;/, "pragma solidity 0.8.13;"); print}
 source line number 1
awk: unknown option -i ignored

awk: can't open file {gsub(/pragma solidity 0\.8\.10;/, "pragma solidity 0.8.13;"); print}
 source line number 1
awk: unknown option -i ignored
```

I also get this error:

```
[⠢] Compiling...
[⠆] Compiling 93 files with 0.8.10
[⠊] Compiling 60 files with 0.8.15
[⠆] Solc 0.8.10 finished in 17.30s
[⠔] Solc 0.8.15 finished in 21.76s
Error: 
Compiler run failed:
Error: Yul exception:Cannot swap Slot JUNK with Slot RET: too deep in the stack by 1 slots in [ JUNK RET[read_from_storage_split_offset_uint256] _2 expr_26607_component_1_mpos expr_26573_address RET[finalize_allocation] var_ins_mpos _6 _4 var_self_26502_slot _5 _7 _3 _1 _8 var_i expr_11 RET ]
```

Reply:
Ah okay, I am not seeing those warnings. It must be a cross platform issue. The second error is a result of not having an updated pragma so it should not happen once we get the script working on mac.

Reply:
Looks like the issue was inplace. Let me know if this works.

https://github.com/ScopeLift/flexible-voting-clients/pull/1/commits/2e58367bc0af4e603ec5999e9dbb7f5c802c02c3